juwai/ansible-role-common
# plat-544 Amazon Linux reverts timezone changes when running yum update

## Summary of changes

check `/etc/sysconfig/clock` and then modified `zone`

## How to Test

1. you need test on amazon linux
2. run ansible for amazon linux
3. check server date `date`

## Note
https://serverfault.com/questions/445984/prevent-amazon-ec2-time-zone-from-reverting-back-on-yum-update

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/set-time.html

https://forums.aws.amazon.com/thread.jspa?threadID=76293

/fyi:  @HouCoder ,  @mark-juwai : Please review.
